### PR TITLE
Clean up wool code

### PR DESCRIPTION
**Pull request by HybridDog**

### CODE REVIEW

**Comment by tenplus1** [comment]
+1 good idea, and personally I would remove the 'if craft_color_group then' check as well.

**Comment by HybridDog** [comment]
>+1 good idea

thanks
>personally I would remove the 'if craft_color_group then' check as well.

l forgot removing this, l did it now and l also removed useless and even wrong (group:wool is not white wool) comments.

**Comment by HybridDog** [comment]
>+1 good idea

thanks
>personally I would remove the 'if craft_color_group then' check as well.

l forgot removing this, l did it now and l also removed useless and even wrong (group:wool is not white wool) comments. @CasimirKaPazi

File: mods/wool/init.lua [hunk 1]
```diff
@@ -26,25 +20,28 @@ wool.dyes = {
...
 
-for _, row in ipairs(wool.dyes) do
-	local name = row[1]
-	local desc = row[2]
-	local craft_color_group = row[3]
-	-- Node Definition
-	minetest.register_node("wool:"..name, {
-		description = desc.." Wool",
-		tiles = {"wool_"..name..".png"},
+for i = 1, #wool.dyes do
```
Comment (t4im) [inline mods/wool/init.lua]: You removed the `wool` table though ;-)

Comment (HybridDog) [inline mods/wool/init.lua]: fixed, l hope that gets merged:
https://github.com/minetest/minetest_game/pull/1165

**Comment by t4im** [comment]
looks good :+1:

**Comment by sofar** [comment]
You remove the `wool.dyes` global - for what reason?

**Comment by sofar** [comment]
NVM it wasn't global to begin with.

**Comment by paramat** [comment]
:+1: if tested.

**Comment by paramat** [comment]
+1 for the first commit.

**Comment by HybridDog** [comment]
>With repeated optimisng there's a chance of gradual reduction in quality.

the quality isn't affected, it's lossless png recompression

however, if the texture files are changed, git stores the old and new ones, so the repository size increases and cloning takes more time, l'm not sure if there's a way to remove unused obsolete files from history without rebasing it

l put it into a second commit on purpose, so you can simply take only the first commit.

**Comment by paramat** [comment]
Ok :+1: for both.

**Comment by paramat** [comment]
@sofar do you +1 both commits?

**Comment by sofar** [comment]
Tested. :+1:

The size saved due to PNG optimization is really, really minimal though. For some of the images, only *3* bytes were saved (on 200+, so not even 1.5%). Oh well.

**Comment by paramat** [comment]
15b82f5b03228f0ded4760ed17934a726ca0003b
1231f726280decf506d36a25d5494302ebb31818

**Pull request closed without merging**